Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -474,13 +474,13 @@ def extract_created_date(entry_list: list):
     return None
 
 
-def get_prediction_single_url(model, url, force_model, who_is_enabled, debug):
+def get_prediction_single_url(model, url, force_model, who_is_enabled, debug, rasterize_timeout):
```

Comment:
```suggestion
def get_prediction_single_url(model, url, force_model, who_is_enabled, debug, rasterize_timeout=TIMEOUT_RASTERIZE):
```

Reply:
The value for the argument *rasterize_timeout* is decided by:
`arg_to_number(demisto.args().get('rasterize_timeout', TIMEOUT_RASTERIZE))`
in the main, so I do not think it is necessary here